packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -226,7 +214,7 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
   ]);
 
   const defaultOnRenderVideoTile = useCallback(
-    (participant: VideoGalleryRemoteParticipant, isVideoParticipant?: boolean, style?: CSSProperties) => {
+    (participant: VideoGalleryRemoteParticipant, isVideoParticipant?: boolean) => {
```

Comment:
Not for this PR: `defaultOnRenderVideoTile` should be a private free function instead of a `useCallback`.

We call it directly via `Array.map`, so there is no point memoizing to get to a fixed number of arguments.